primitives/src/locktime/relative.rs

```diff
@@ -89,6 +112,16 @@ impl LockTime {
     ///
     /// This method will **not** round-trip with [`Self::to_sequence`]. See the
     /// docs for [`Self::from_consensus`] for more information.
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use bitcoin_primitives::{Sequence, relative::LockTime};
+    ///
+    /// // Interpret a sequence number from a Bitcoin transaction input as a relative lock time
+    /// let sequence_number = Sequence::from_consensus(7000);
+    /// let lock_time = LockTime::from_sequence(sequence_number).unwrap();
+    /// assert!(lock_time.is_block_height());
```

Comment:
Why 7000? This is eventually just the low 16 bits, right? Would it be better to use 100 like in the other example?

Reply:
7000 looks to me like it was supposed to be hex but we forget the `0x` so it makes me wonder.

I'm wondering if for all these examples we should use some 'sane' arbitrary value. For example we could use approximately 24 hours which is

- 168 time intervals
- 144 blocks (mathematically incorrect but everyone uses this fudge, for the real distribution see https://en.wikipedia.org/wiki/Poisson_distribution)

Reply:
I changed these and added comments.